datasets/pre-post-purchase-questions.yml

```diff
@@ -0,0 +1,25 @@
+Name: Pre- and post-purchase product questions
+Description: This dataset provides product related questions, including their textual content and gap, in hours, between purchase and posting time. 
+Each question is also associated with related product details, including its id and title.
+Documentation: https://pre-post-purchase-questions.s3.amazonaws.com/README.txt
+Contact: litalku@amazon.com
+ManagedBy: "[Amazon](https://www.amazon.com/)"
+UpdateFrequency: Not currently being updated
+Tags:
+  - aws-pds
+  - amazon.science
+  - natural language processing
+  - machine learning
+License: "[Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0)"
+Resources:
+  - Description: S3 bucket with dataset
+    ARN: arn:aws:s3:::pre-post-purchase-questions
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore:
+      - "[PrePostQuestions.csv](https://pre-post-purchase-questions.s3.amazonaws.com/PrePostQuestions.csv)"
+DataAtWork:
+  Publications:
+    - Title: "Did you buy it already?", Detecting Users Purchase-State From Their Product-Related Questions
+      URL:
```

Comment:
Do you have a URL that can be used for this? You can change it later. Or we can remove the publication until we have the URL. Otherwise this should be ready to go live if you are ready

Reply:
this is the URL for now: https://sigir.org/sigir2021/